go.mod

```diff
@@ -29,6 +28,7 @@ require (
 	github.com/x448/float16 v0.8.4 // indirect
 	golang.org/x/crypto v0.31.0 // indirect
 	golang.org/x/mod v0.22.0 // indirect
+	golang.org/x/net v0.33.0 // indirect
```

Comment:
Could you try removing this indirect import and `go mod tidy` again?

Reply:
Updated.